apps/editor/src/app/locales/en.json

```diff
@@ -1275,7 +1282,13 @@
         "passwordToken": "How long (in minutes) the link sent via email to reset password will be valid. Defaults to one day (1440 / 60 / 24).",
         "peerToken": "Timeout in milliseconds after which we will no longer try to pull peer articles. Defaults to 3 seconds.",
         "invoiceReminders": "How many times we will remind user to pay the subscription (once per day).",
-        "askOperators": "Warning: changing one or more of the settings may break the page or have other unpredictable consequences. If you are unsure, please try talking to the operators first. If you are sure, feel free to continue."
+        "askOperators": "Warning: changing one or more of the settings may break the page or have other unpredictable consequences. If you are unsure, please try talking to the operators first. If you are sure, feel free to continue.",
+        "newSubscriptionsApiPublic": "Exposes subscriptions anonymized to the public.",
+        "activeSubscriptionsApiPublic": "Exposes subscriptions anonymized to the public.",
+        "renewingSubscriptionsApiPublic": "Exposes subscriptions anonymized to the public.",
+        "newDeactivationsApiPublic": "Exposes subscriptions anonymized to the public.",
```

Comment:
These all meant to have the same text? Should each one have something more descriptive?

Reply:
No they are intended to be like this.